pkg/simple/client/auditing/elasticsearch/options.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 type Options struct {
+	Enabled     bool   `json:"enabled" yaml:"enable"`
```

Comment:
Why json tag is `enabled` while yam tag is `enable`, this is misleading. I think `enable` will be just fine.